modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -31,6 +31,7 @@
   "changesAwaitingApproval": "Changes to this document are awaiting approval by an admin or editor.",
   "changesDiscarded": "Changes discarded",
   "changesPublished": "Your changes have been published.",
+  "cannotChangeSlugPrefix": "cannot change the slug prefix",
```

Comment:
ha i meant this as a placeholder, this probably needs some softening and context at some point